packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,136 @@
+# Capturing Vital Signs
+
+Vital signs are a group of important medical signs that measure the body's most vital (life-sustaining) functions. The four main vital signs are body temperature, pulse rate, respiration rate, and blood pressure. However, sometimes other measures, such as blood oxygen, weight, and blood glucose level, are included as well.
+
+## The `Obeservation` Resource
+
+Vital signs are stored as [`Observation`](/docs/api/fhir/resources/observation) resources. To track the specific vital that is being observed, use the `Observation.code` element. This describes what was observed, and is sometimes called the observation &quot;name&quot;. This should be coded using [LOINC](https://loinc.org/search) codes per US core guidelines.
+
```

Comment:
Add a link to the Medplum LOINC https://www.medplum.com/docs/careplans/loinc instead of the LOINC.org site